utils/controleval.py

```diff
@@ -108,14 +130,72 @@ def print_specific_stat(stat, current, total):
 def stats(ctrlmgr, args):
     validate_args(ctrlmgr, args)
     ctrls = set(ctrlmgr.get_all_controls_of_level(args.id, args.level))
-    calculate_stats(ctrls)
+    if args.output_json_file is not None:
+        print_to_json(
+            ctrls,
+            args.product,
+            args.id,
+            args.level,
+            args.output_json_file)
+    else:
+        calculate_stats(ctrls)
+
+
+def print_to_json(ctrls, product, id, level, output_file):
+    total = len(ctrls)
+    data = dict()
+    ctrllist = collections.defaultdict(set)
+
+    if total == 0:
```

Comment:
This doesn't seem JSON-specific, perhaps we move it up to the stats() method?

Reply:
thanks that makes sense